### PR TITLE
LPS-193598 Add Autom. Test ClayAccessibilty#AccessibilitySmoke

### PR DESCRIPTION
**Pull request by eltonccdantas**
[Ticket](https://liferay.atlassian.net/browse/LPS-193598)

Given new content page 
And Clay Sample Portlet 
When access all clay sample portlet tabs 
Then axe automation finds no accessibility issues
// use poshi function AssertAccessible

Some tabs were taken out by the test intentionally. They will be included when [LPS-198580](https://liferay.atlassian.net/browse/LPS-198580) is done.

### CODE REVIEW

**Comment by eltonccdantas** [comment]
ci:test:sf

**Comment by eltonccdantas** [comment]
ci:help

**Comment by eltonccdantas** [comment]
ci:test:clay

**Comment by eltonccdantas** [comment]
image

**Comment by john-co** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAccessibility.testcase [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "Clay Sample Test Page");
+		}
+	}
+
+	@description = "LPS-193598. Verifies that clay portlet tabs are accessible."
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAccessibility.testcase:48]: Verifies clay sample portlet components are compliant with axe accessibility

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAccessibility.testcase [hunk 2]
```diff
@@ -0,0 +1,68 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property osgi.modules.includes = "frontend-taglib-clay-sample-web";
+	property portal.accessibility = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Clay";
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAccessibility.testcase:8]: use `property testray.component.names = "PEDS Accessibility";`

**Review by john-co — CHANGES_REQUESTED** [review]
Some minor edits. 

Also, please check on sf test to see if the comments pass. Usually we need to give a reason why a comment is needed and SF tries to catch it.

Besides that looks good to me! :sunglasses:

**Review by john-co — APPROVED** [review]

**Comment by john-co** [comment]
ci:test:sf

**Comment by eltonccdantas** [comment]
ci:test:sf

**Comment by eltonccdantas** [comment]
ci:help

**Comment by eltonccdantas** [comment]
ci:test:peds-accessibility

**Comment by john-co** [comment]
Looks good to me! :sunglasses:

We need to fix the PQL for peds-accessibility test suite...that's way too many tests running

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**